### DISCUSSION TITLE
Artifacts on DisplayLink monitors

### DISCUSSION THREAD

**Discussion by Irubataru** [hyprwm/Hyprland#8381] [Q&A]
I am trying to get DisplayLink to work (unfortunately that is my docking situation). And I do get the monitors to show up, and things do in general work, but I am having issues with artifacts, and I was wondering if there is any way to try to fix that,

**Comment by nktnet1** [comment]
Potentially related to https://github.com/hyprwm/Hyprland/issues/6935
